### PR TITLE
Export YGInteropSetLogger method

### PR DESCRIPTION
**Pull request by KimDaeWook**
When building and using C # libraries,
EntryPointNotFoundException thrown from YGInteropSetLogger.

so, I added YOGA_EXPORT on YGInteropSetLogger.

### CODE REVIEW

**Comment by KimDaeWook** [comment]
And YGConfigGetUseLegacyStretchBehaviour is not defined, so i add it additionally.

File: yoga/Yoga.cpp [hunk 1]
```diff
@@ -4363,6 +4363,11 @@ YOGA_EXPORT void YGConfigSetUseWebDefaults(
   config->useWebDefaults = enabled;
 }
 
+YOGA_EXPORT bool YGConfigGetUseLegacyStretchBehaviour(
```
Comment (SidharthGuglani) [inline yoga/Yoga.cpp]: Do you really need this ? This flag might be removed in future.

**Comment by SidharthGuglani** [comment]
Thanks for creating this PR.
Will merge the PR for adding YOGA_EXPORT to YGInteropSetLogger after you remove other changes

**Comment by NickGerleman** [comment]
It looks like the C# bindings already assume `YGConfigGetUseLegacyStretchBehaviour`. From recent discussions on RN compatibility, I don't think we will be able to remove the flag soon, so adding a public accessor seems righteous to me.

Otherwise, fixing the mismatch in export declaration is a good change.

**Review by NickGerleman — APPROVED** [review]

**Pull request closed without merging**